build.sbt

```
@@ -35,6 +35,10 @@ lazy val root = project
       "io.flow" %% "lib-usage-play28" % "0.1.15",
       "org.typelevel" %% "cats-core" % "2.6.1",
       "org.yaml" % "snakeyaml" % "1.29",
+      "com.datadoghq" % "dd-trace-api" % "0.85.0",
+      "com.datadoghq" % "dd-trace-ot" % "0.85.0",
+      "io.opentracing" % "opentracing-api" % "0.33.0",
+      "io.opentracing" % "opentracing-util" % "0.33.0",
```

```suggestion
      "com.datadoghq" % "dd-trace-ot" % "0.85.0",
```

I think this is enough